src/Microsoft.AspNet.Mvc.Razor/TagHelperContentWrapperTextWriter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.Razor
+{
+    /// <summary>
+    /// <see cref="TextWriter"/> implementation which writes to a <see cref="TagHelperContent"/> instance.
+    /// </summary>
+    public class TagHelperContentWrapperTextWriter : TextWriter
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TagHelperContentWrapperTextWriter"/> class.
+        /// </summary>
+        /// <param name="encoding">The <see cref="Encoding"/> in which output is written.</param>
```

Comment:
this approximates how the `Encoding` property in `TextWriter` is documented.  as you say, it is part of the class' requirements.  the value should match the rest of our `TextWriter`s so future uses (there are very few today) work correctly.

in any case, I'm not revamping `TagHelperContentWrapperTextWriter` in this way as part of this PR.  this is for Beta4.